tests/storages_tests/test_heartbeat.py

```diff
@@ -22,6 +22,21 @@
 from optuna.trial import TrialState
 
 
+@pytest.mark.parametrize("storage_mode", STORAGE_MODES_HEARTBEAT)
+def test_repeatedly_called_record_heartbeat(storage_mode: str) -> None:
+    with StorageSupplier(storage_mode) as storage:
+        if not is_heartbeat_enabled(storage):
+            return
```

Comment:
I think `pytest.skip` or `assert` is more appropriate here.
Actually when does this test will be skipped?
If this test is always skipped, it does not make sense to have this test at all because this unittest has only one parameter with one choice.